lib/cli.ts

```diff
@@ -87,14 +87,10 @@ async function run() {
   program
     .command("init")
     .description("Initialize a new project")
-    .argument("[path]", "where to init the project")
-    .option("-t, --template <template>", "template to use (js-single, js-multi)", "js-single")
-    .action(async (path, opts) => {
-      if (!path) {
-        log.error(`[path] argument is required`);
-        log.log(`Usage example: init ./example\n`);
-      }
-      await initProject(path, opts.template);
+    .option("-p, --path <path>", "where to init the project", ".")
```

Comment:
This all looks good, can you run `bw help` and update the README with this new command?